lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -176,6 +176,12 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
     @Input()
     sidebarLeftTemplateContext: T = null;
 
+    /**
+     * Change the close button position Right/Left.
+     */
+    @Input()
+    closeButtonPosition = 'right';
```

Comment:
Could you please narrow down type to allow only 'right' or 'left' values? Actually typescript sees that you assign string to it sets 'string' type for closeButtonPosition so somebody can pass some incorrect string like 'abc'. We should set type here which allows to use only right and left values.